docs/developers/infrastructure.md

```diff
@@ -0,0 +1,42 @@
+# Infrastructure Setup
+The whole infrastructure of data package registry is built on AWS services.
+## Domain and certificate setup:
+AWS has domain hosting service called ```Route53```. This is the only place we are not using aws service.
+
+The extrnal domain: **datapackaged.com**
+
+We mainly have 2 parts:
+- ApiGateWay [Custom domain]
+- S3
+    - Static Files [static.datapackages.com]
+    - Data File [bits.datapackaged.com]
+
+### Api Gateway:
+For api gateway having custom domain we have to add generated
+- Certificate body
+- Certificate private key
+- Certificate chain
+We use ```zerossl``` to generate SSL certificate.
+The process is described here:
+https://www.pandastrike.com/posts/20160613-ssl-cert-aws-api-gateway-zerossl-letsencrypt
+
+Once we add the certs in ApiGateway it will gevetare cloudfront distribution.
```

Comment:
Would use `certificates` instead of `certs`

Also Typo? 
*gevetare -> generate